plugins/analysis-nori/build.gradle

```diff
@@ -32,6 +32,9 @@ restResources {
     includeCore '_common', 'indices', 'index', 'search'
   }
 }
+
+integTest.enabled = false
```

Comment:
Ditto. If we need to disable a test, can we have a comment explaining why it is disabled? Someone coming along in the future will have no reason whether this can/should be re-enabled.

Reply:
This isn't really disabling a test ... it is to avoid the testing conventions from complaining.  I would prefer to tweak the testing conventions instead of a lot of redunant comments. I can do that in a separate PR (and remove these in this PR)